oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/endpoint/OAuth2PasswordGrantRequestEntityConverterTests.java

```diff
@@ -110,9 +110,14 @@ public void convertWhenParametersConverterSetThenCalled() {
 	@SuppressWarnings("unchecked")
 	@Test
 	public void convertWhenGrantRequestValidThenConverts() {
-		ClientRegistration clientRegistration = TestClientRegistrations.password().build();
+		ClientRegistration clientRegistration = TestClientRegistrations.password()
+			.clientId("clientId")
+			.clientSecret("clientSecret=")
+			.build();
```

Comment:
Since the `convertWhenGrantRequestValidThenConvertsWithoutUrlEncoding` is the test that confirms the URL encoding toggle works, it felt like the correct move would be to include a slightly change on an earlier test that confirms the URL encoding default was enabled.

This felt better than copying and pasting an entire test just to add in a `=`, without any other changes. Where as the `convertWhenGrantRequestValidThenConvertsWithoutUrlEncoding` is a copy+paste of another test with the `=` added, and the toggle swapped.

Reply:
Likewise the historical tests + this minor change mean the class has coverage, which sure is a little bit of a weak argument, but adding in several tests for this feels a bit much.